.github/workflows/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: go-workflow
+on:
+  push:
+    # Temporarily removed for first PR
+    # branches:
+    #   - master
+  pull_request:
+jobs:
+  golang-tests:
+    name: tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        go-version: ["1.15", "1.14", "1.13"]
+        go111module: ["on", "off"]
+        include:
+          # includes goflags when go111module is on
+          - go111module: "on"
+            goflags: -mod=readonly
+    env:
+      GO111MODULE: ${{ matrix.go111module }}
+      GOPATH: ${{ github.workspace }}
```

Comment:
Is the `GOPATH` required for both modes?

Reply:
Only important for `GO111MODULE=on`

Reply:
To my surprise, it is necessary for `go get` and `go build`.

It must be because of:
```
defaults:
      run:
        working-directory: ${{ github.workspace }}/src/github.com/getsentry/sentry-go
```

Are you OK if we leave `GOPATH` in place? Alternatively we can separate the flows for module on vs off to avoid all this custom steps.